apps/services/auth/admin-api/project.json

```diff
@@ -45,7 +45,12 @@
     "dev": {
       "executor": "nx:run-commands",
       "options": {
-        "command": "yarn start --project services-auth-admin-api"
```

Comment:
The dev target is being deduplicated, but I prefer using `command` for one command instead of the more verbose `commands` array.